<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCDProGui</class>
 <widget class="QWidget" name="FCDProGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>394</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>443</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>FunCubeDongle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutFrequency">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="deviceUILayout">
       <item>
        <layout class="QHBoxLayout" name="deviceButtonsLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop acquisition</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrgui/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deviceRateLayout">
         <item>
          <widget class="QLabel" name="deviceRateText">
           <property name="toolTip">
            <string>I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Liberation Mono</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Tuner center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnits">
       <property name="text">
        <string> kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutPPM">
     <item>
      <widget class="QLabel" name="ppmLabal">
       <property name="text">
        <string>LO ppm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ppm">
       <property name="toolTip">
        <string>Local Oscillator ppm correction</string>
       </property>
       <property name="minimum">
        <number>-2000</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ppmText">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="corrLabel">
       <property name="text">
        <string>Auto corr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="dcOffset">
       <property name="toolTip">
        <string>DC offset auto correction</string>
       </property>
       <property name="text">
        <string>DC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="iqImbalance">
       <property name="toolTip">
        <string>IQ imbalance auto correction</string>
       </property>
       <property name="text">
        <string>IQ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="fcPosLabel">
       <property name="text">
        <string>Fp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fcPos">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Relative position of device center frequency</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Inf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sup</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cen</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDecim">
       <property name="text">
        <string>Dec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="decim">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decimation factor</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="TransverterButton" name="transverter">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Transverter frequency translation dialog</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBiasT">
     <item>
      <widget class="QPushButton" name="setDefaults">
       <property name="toolTip">
        <string>Set all values to typical defaults</string>
       </property>
       <property name="text">
        <string>Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBand">
     <item>
      <widget class="QLabel" name="lnaEnhanceLabel">
       <property name="text">
        <string>LNA Enh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lnaEnhance">
       <property name="toolTip">
        <string>LNA enhancement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bandLabel">
       <property name="text">
        <string>Band</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="band">
       <property name="toolTip">
        <string>Band optimizations</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBias">
     <item>
      <widget class="QLabel" name="biasLabel">
       <property name="text">
        <string>Bias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bias">
       <property name="toolTip">
        <string>Band bias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mode">
       <property name="toolTip">
        <string>Mode ?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutLNA">
     <item>
      <widget class="QLabel" name="lnaGainLabel">
       <property name="text">
        <string>LNA G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lnaGain">
       <property name="toolTip">
        <string>LNA gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rfFilterLabel">
       <property name="text">
        <string>RF filt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rfFilter">
       <property name="toolTip">
        <string>RF filter selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMixer">
     <item>
      <widget class="QLabel" name="mixGainLabel">
       <property name="text">
        <string>Mix G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mixGain">
       <property name="toolTip">
        <string>Mixer gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mixFilterLabel">
       <property name="text">
        <string>Mix filt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mixFilter">
       <property name="toolTip">
        <string>Mixer filter selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutGain1">
     <item>
      <widget class="QLabel" name="gain1Label">
       <property name="text">
        <string>Gain1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gain1">
       <property name="toolTip">
        <string>Gain block #1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rcFilterLabel">
       <property name="text">
        <string>RC filt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rcFilter">
       <property name="toolTip">
        <string>Baseband RC filter selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutGain2">
     <item>
      <widget class="QLabel" name="gain2Label">
       <property name="text">
        <string>Gain2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gain2">
       <property name="toolTip">
        <string>Gain block #2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gain3Label">
       <property name="text">
        <string>Gain3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gain3">
       <property name="toolTip">
        <string>Gain block #3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutGain4">
     <item>
      <widget class="QLabel" name="gain4Label">
       <property name="text">
        <string>Gain4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gain4">
       <property name="toolTip">
        <string>Gain block #4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IF filt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ifFilter">
       <property name="toolTip">
        <string>IF filter selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutGain5">
     <item>
      <widget class="QLabel" name="gain5Label">
       <property name="text">
        <string>Gain5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gain5">
       <property name="toolTip">
        <string>Gain block #5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gain6Label">
       <property name="text">
        <string>Gain6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gain6">
       <property name="toolTip">
        <string>Gain block #6</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>TransverterButton</class>
   <extends>QPushButton</extends>
   <header>gui/transverterbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
